docs/building-a-form/common-definitions.md

```diff
@@ -65,37 +65,59 @@ A common type-ahead widget that lets a user type in values and narrow down a lon
 
 ### Bank account
 
-The common electronic funds transfer (EFT) account information field that collects account type (checking or savings), bank account number, and bank routing number.
+The common electronic funds transfer (EFT) account information field that collects account type (checking or savings), bank account number, and bank routing number. Use these definitions as the uiSchema for the field that represents the account number.
 
-- File: [/src/js/definitions/bankAccount.js](../../src/js/definitions/bankAccount.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Object
+```js
+import { uiSchema as bankAccountUI }
```

Comment:
This has tripped me up before, but I'm pretty sure using the curly braces in the import statement here will cause an issue because there's a default export in `bankAccount.js`. I think instead the import statement should be `import bankAccountUI from 'us-forms-system/lib/js/definitions/bankAccount';`.